lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -9,19 +9,31 @@ class FirebaseAppDistributionApiClient
       MAX_POLLING_RETRIES = 60
       POLLING_INTERVAL_SECONDS = 2
 
+      # Enables tester access to the specified app release. Skips this
+      # step if no testers are passed in (emails and group_ids are nil).
+      #
+      # args
+      #   app_id - Firebase App ID
+      #   release_id - App release ID, returned by upload_status endpoint
+      #   emails - String array of app testers' email addresses
+      #   group_ids - String array of Firebase tester group IDs
+      #
+      # Throws a user_error if app_id, emails, or group_ids are invalid
       def enable_access(app_id, release_id, emails, group_ids)
         if emails.nil? && group_ids.nil?
```

Comment:
Consider checking if `emails` and `groups` are empty as well (and adding a test 🙂)